appintro/src/main/java/com/github/paolorotolo/appintro/internal/LayoutUtil.kt

```diff
@@ -11,7 +11,11 @@ internal object LayoutUtil {
 
     @JvmStatic
     fun isRtl(ctx: Context): Boolean {
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 &&
-                ctx.resources.configuration.layoutDirection == View.LAYOUT_DIRECTION_RTL
+       var isRTL : Boolean
+        isRTL = false //RTL is supported only on API 17, so it will be false on API<17
```

Comment:
This is equivalent to before. Can you please revert?

Reply:
Thanks. I learnt about short circuit evaluation because of this. I will run it on an API 16 device and report if I get any errors. Resolving for now.